Snowflake.Data.Tests/SFPutGetTest.cs

```diff
@@ -62,13 +61,19 @@ public void TestPutCommand(string compressionType)
                 // Write row data to temp file
                 File.WriteAllText(filePath, ROW_DATA);
 
-                string putQuery = $"PUT file://${filePath} @{DATABASE_NAME}.{SCHEMA_NAME}.%{TEST_TEMP_TABLE_NAME}";
+                string tempDirectory = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
```

Comment:
PUT and GET test should cover PUT/GET to stage other than table. Also compression option tests are missing. If these are not addressed in this PR, PR summary should be updated accordingly.

Reply:
I've added tests for the other staging types and updated the summary